### PR TITLE
Introduce Python black to all files.

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: bedrock/base/log_settings.py [hunk 1]
```diff
@@ -14,89 +14,86 @@ def emit(self, record):
         pass
 
 
-base_fmt = ('%(name)s:%(levelname)s %(message)s '
-            ':%(pathname)s:%(lineno)s')
+base_fmt = "%(name)s:%(levelname)s %(message)s " ":%(pathname)s:%(lineno)s"
```
Comment (robhudson) [inline bedrock/base/log_settings.py:17]: FIXME

File: bedrock/base/middleware.py [hunk 2]
```diff
@@ -87,20 +89,18 @@ def __call__(self, request):
...
                         if provided_auth == required_auth:
                             # we're good. continue on.
                             return None
 
-            response = HttpResponse(status=401,
-                                    content='<h1>Unauthorized. '
-                                            'This site is in private demo mode.</h1>')
-            realm = settings.APP_NAME or 'bedrock-demo'
-            response['WWW-Authenticate'] = 'Basic realm="{}"'.format(realm)
+            response = HttpResponse(status=401, content="<h1>Unauthorized. " "This site is in private demo mode.</h1>")
```
Comment (robhudson) [inline bedrock/base/middleware.py:101]: FIXME

File: bedrock/base/tests/test_accepted_locales.py [hunk 3]
```diff
@@ -81,10 +83,10 @@ def test_dev_languages(self):
         settings.DEV = True
         # simulate the successful result of the DEV_LANGUAGES list
         # comprehension defined in settings.
-        settings.DEV_LANGUAGES = ['en-US', 'fr']
-        assert settings.LANGUAGE_URL_MAP == {'en-us': 'en-US', 'fr': 'fr'}, (
-            'DEV is True, but DEV_LANGUAGES are not used to define the '
-            'allowed locales.')
+        settings.DEV_LANGUAGES = ["en-US", "fr"]
+        assert settings.LANGUAGE_URL_MAP == {"en-us": "en-US", "fr": "fr"}, (
+            "DEV is True, but DEV_LANGUAGES are not used to define the " "allowed locales."
```
Comment (robhudson) [inline bedrock/base/tests/test_accepted_locales.py:88]: FIXME

File: bedrock/contentcards/tests/test_models.py [hunk 4]
```diff
@@ -8,98 +8,96 @@
...
-            }
+                "title": "We keep your data safe, never sold.",
+                "ga_title": "We keep your data safe, never sold.",
+                "highres_image_url": "/media/contentcards/img/home/ffyr-high-res.191bff93b820.png",
+                "media_icon": "mzp-has-video",
+                "class": "mzp-c-card-large",
+                "image_url": "/media/contentcards/img/home/ffyr.75c74c6ba409.png",
+                "youtube_id": "rZAQ6vgt8nE",
+                "aspect_ratio": "mzp-has-aspect-16-9",
+                "desc": "You have the right to your own life \u2014 and your own data. " "Everything we make and do fights for you.",
```
Comment (robhudson) [inline bedrock/contentcards/tests/test_models.py:69]: FIXME

File: bedrock/base/tests/test_accepted_locales.py [hunk 5]
```diff
@@ -93,6 +95,4 @@ def test_prod_languages(self):
 
         """
         settings.DEV = False
-        assert settings.LANGUAGE_URL_MAP == {'en-us': 'en-US'}, (
-                'DEV is False, but PROD_LANGUAGES are not used to define the '
-                'allowed locales.')
+        assert settings.LANGUAGE_URL_MAP == {"en-us": "en-US"}, "DEV is False, but PROD_LANGUAGES are not used to define the " "allowed locales."
```
Comment (robhudson) [inline bedrock/base/tests/test_accepted_locales.py:98]: FIXME

**Review by pmac — APPROVED** [review]
🍻

**Pull request merged**